lib/src/main/java/io/fabianterhorst/isometric/Isometric.java

```diff
@@ -390,7 +369,7 @@ public Item findItemForPosition(Point position, boolean reverseSort, boolean tou
         Point[] transformedPoints;
         android.graphics.Path drawPath;
 
-        Item(Item item) {
+        public Item(Item item) {
```

Comment:
Maybe add an static creating method that makes it easier for the user to create an item without manually using the transformColor method.